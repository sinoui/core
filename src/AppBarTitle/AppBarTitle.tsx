import styled from 'styled-components';

/**
 * 顶部应用栏标题
 */
const AppBarTitle = styled.h1.attrs(() => ({
  className: 'sinoui-app-bar--title',
}))((props) => ({
  ...props.theme.typography.h6,
  margin: 0,
  overflow: 'hidden',
}));

export default AppBarTitle;
