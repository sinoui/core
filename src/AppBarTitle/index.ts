import AppBarTitle from './AppBarTitle';

export default AppBarTitle;
