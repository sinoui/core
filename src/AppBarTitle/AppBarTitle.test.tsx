import React from 'react';
import { ThemeProvider } from 'styled-components';
import { defaultTheme } from '@sinoui/theme';
import renderer from 'react-test-renderer';
import AppBarTitle from '@sinoui/core/AppBarTitle';

/**
 * 应用栏 标题组件 单元测试
 */

it('AppBarTitle', () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={defaultTheme}>
        <AppBarTitle>标题</AppBarTitle>
      </ThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
