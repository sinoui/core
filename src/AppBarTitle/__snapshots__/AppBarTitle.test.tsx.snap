// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBarTitle 1`] = `
<h1
  className="AppBarTitle-sc-1lmkalg-0 hVbawm"
>
  标题
</h1>
`;
